lib/emittr/events.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'setup'
```

Comment:
Shouldn't we be including `Emittr::Setup` in this class instead of in the `Emitter`? It sounds weird to me to separate it in two modules if they coupled.